tyssue/core/objects.py

```diff
@@ -59,8 +59,12 @@
         }
 '''
 
-
-class Epithelium:
+#- BC -#
+# Classes must inherit from Object in order for setters
+# decorators to work properly according to some
+# stackoverflow post, but it seems to work properly as
+# it is right now.
+class Epithelium():
```

Comment:
You're right, in Python 3.X classes implicitly inherit from object, so nevermind.